app/controllers/foo_controller.rb

```diff
@@ -9,4 +9,27 @@ def is_string?(string)
       'stringじゃないよ'
     end
   end
+
+  def long_method
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->